### ISSUE TITLE
Picture in picture mode

### ISSUE THREAD

**Issue by vitaliybaran** [mynaparrot/moodle-mod_plugnmeet#23]
When I choose a Picture in Picture mode and a little screen appears above all other tabs, the video in the little screen freezes after some seconds and when I get back to the original sessiob tab, the little screen is live again. It freezes over all the tabs. It works only over the original tab. Is it only the case with my moodle plugin?
I use ready-to-use [plugNmeet cloud solution](https://www.plugnmeet.cloud/) api. free account.

**Comment by jibon57** [comment]
Are you using virtual background during share webcam? If so then it's an expected behavior of browser.

**Comment by vitaliybaran** [comment]
This happens both in Google chome and MSEdge. I use these two.

**Comment by jibon57** [comment]
Is this happening for your own video or other users video? You can try to disable with Dynacast feature.

**Comment by wbobteng** [comment]
Hello,i think this is not related with plugin, it should related with browser or devices. I suggest you to try with another devices. Meanwhile, for picture in Picture feature, you should be able to move the small screen to anywhere on your screen.

**Comment by vitaliybaran** [comment]
Yes, the video freezes with other users only. My own video works perfectly well. I checked it on two PCs (Windows 10 Home). It's the same in both cases.

**Comment by vitaliybaran** [comment]
P.S. One more thing is not quite logical, I think. It's the timer. Although the timer is VERY useful, it would be great if it started off only when the admin logs in and starts the session. Be cause now it is triggered with the login of any user. For example, if I choose the lesson to last for say 60 mins and one of my pupils decides to log in 10 mins ealier and just wait for me to accept him (I always use the "waiting room" feature), the timer is triggered with the first student's login, so very ofen when I start the lesson on time, the duration of the session is not 60 mins any more, but 50.

**Comment by jibon57** [comment]
Nope, we want to keep logic based on meeting created.

**Comment by vitaliybaran** [comment]
But if a student wishes to start waiting 10 or 15 mins earlier just not to get late for the class, everybody suffers. It is the teacher who should be in control of the meeting duration not the student. I can't possibly make a kid start the meeting at the exact time as me. Hence the logic: if the waiting room feature is enabled, then techically if ones student chooses to wait for some time he/she should be the one to decide on the duration of the class.

**Comment by jibon57** [comment]
Your application should have ability to disable early joining. Duration of session also affect server bandwidth. Some people hire server based on duration too. So it will effect on those too. I'll recommend you to configure in your application.
